src/shared/seccomp-util.c

```diff
@@ -523,6 +523,12 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "keyctl\0"
                 "request_key\0"
         },
+        [SYSCALL_FILTER_SET_SYSLOG] = {
+                .name = "@syslog",
+                .help = "Kernel message ring buffer",
+                .value =
+                "syslog\0"
+        },
```

Comment:
hmm, having a group for a single syscall only sounds unnecessary. Let's just block the syscall directly, no need to go via the group